include/eve/detail/function/simd/common/subscript.hpp

```diff
@@ -89,7 +89,7 @@ namespace eve::detail
     {
       // g++ need that
       typename Wide::value_type data[Wide::size()];
-      auto s = p.storage();
+      [[maybe_unused]] auto s = p.storage();
```

Comment:
why?

Reply:
I once compiled by mistake using g++ 13 in release mode and got a warning. I fixed it then realized it was out of our scope. If you want me to remove it I can.